Lib/pdb.py

```diff
@@ -1638,9 +1638,9 @@ def help():
     pydoc.pager(__doc__)
 
 _usage = """\
-usage: pdb.py [-c command] ... pyfile [arg] ...
+usage: pdb.py [-c command] ... [-m module | pyfile] [arg] ...
 
-Debug the Python program given by pyfile.
+Debug the Python program given by pyfile or the module specified.
```

Comment:
I think this will be clearer as two sentences: "Debug the Python program given by pyfile. Alternatively, an executable module or package to debug can be specified using the -m switch."